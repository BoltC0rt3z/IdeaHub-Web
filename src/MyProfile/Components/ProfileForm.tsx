import React from 'react';
import { Row } from 'antd';

import { Input, TextArea } from '../../Components';
import { profileImage, coverImage } from '../../MockData';

import styles from './ProfileForm.module.scss';

const ProfileForm: React.FC = () => {
  return (
    <Row className={styles.profileContainer}>
      <Row>
        <Row className={styles.coverPhotoContainer}>
          <img src={coverImage} alt="Cover" className={styles.image} />
        </Row>
        <Row className={styles.profilePhotoContainer}>
          <img src={profileImage} alt="Profile" className={styles.image} />
        </Row>
      </Row>
      <Row>
        <Input placeholder="Name" />
      </Row>
      <Row className={styles.input}>
        <TextArea placeholder="Bio" />
      </Row>
      <Row className={styles.input}>
        <Input placeholder="Location" />
      </Row>
      <Row className={styles.input}>
        <Input placeholder="Website" />
      </Row>
      <Row className={styles.input}>
        <Input placeholder="Birth date" />
      </Row>
    </Row>
  );
};

export default ProfileForm;
