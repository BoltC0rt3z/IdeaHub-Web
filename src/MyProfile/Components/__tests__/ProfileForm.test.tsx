import React from 'react';
import { mount, ReactWrapper } from 'enzyme';

import ProfileForm from '../ProfileForm';

describe('Tests for profile form', () => {
  const wrapper: ReactWrapper = mount(<ProfileForm />);

  it('Render profile form without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
