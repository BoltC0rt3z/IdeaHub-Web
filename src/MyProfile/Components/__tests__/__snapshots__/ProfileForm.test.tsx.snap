// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for profile form Render profile form without crushing 1`] = `ReactWrapper {}`;
